test/common_test/construct_empty_world.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <gtest/gtest.h>
+
+#include <gz/common/Console.hh>
+
+#include "../helpers/TestLibLoader.hh"
+
+#include <gz/physics/ConstructEmpty.hh>
+#include <gz/physics/FindFeatures.hh>
+#include <gz/physics/GetEntities.hh>
+#include <gz/physics/RequestEngine.hh>
+
+// The features that an engine must have to be loaded by this loader.
+using Features = gz::physics::FeatureList<
+  gz::physics::GetEngineInfo,
+  gz::physics::GetEntities,
+  gz::physics::ConstructEmptyWorldFeature
+>;
+
+class ConstructEmptyWorldTest:
+public gz::physics::TestLibLoader
+{
+  // Documentation inherited
+  public: void SetUp() override
+  {
+    gz::common::Console::SetVerbosity(4);
+
+    auto plugins = loader.LoadLib(ConstructEmptyWorldTest::GetLibToTest());
```

Comment:
I think this `plugins` variable isn't used for anything

Reply:
https://github.com/gazebosim/gz-physics/pull/360/commits/0ecea70111124b5700a9ac45c00ee58102a6ca3b